src/mpid/ch4/shm/ipc/src/ipc_p2p.h

```diff
@@ -272,7 +270,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_IPCI_handle_lmt_recv(MPIDI_IPC_hdr * ipc_hdr,
 
     IPC_TRACE("handle_lmt_recv: handle matched rreq %p [source %d, tag %d, "
               " context_id 0x%x], copy dst %p, bytes %ld\n", rreq,
-              MPIDIG_REQUEST(rreq, u.recv.source), MPIDIG_REQUEST(rreq, u.recv.tag),
```

Comment:
I don't see the fields actually removed from `MPIDIG_req_t` in `mpidpre.h`. Is it supposed to be in this patch?

Reply:
I missed it. I just added the commit.